kubeadm/flannel/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG servercoreTag="ltsc2019"
+ARG cniVersion="0.8.5"
+
+FROM mcr.microsoft.com/windows/servercore:${servercoreTag}
+SHELL ["powershell", "-NoLogo", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ARG cniVersion
+
+# Stuck on a prerelease flannel until https://github.com/coreos/flannel/issues/1231 is resolved
+RUN mkdir C:\flannel; \
+  pushd C:\flannel; \
+  curl.exe -LO https://github.com/benmoss/flannel/releases/download/v0.12.0-rc1/flanneld.exe
+
+RUN mkdir C:\cni; \
+  pushd C:\cni; \
+  curl.exe -Lo cni.tgz https://github.com/containernetworking/plugins/releases/download/v${env:cniVersion}/cni-plugins-windows-amd64-v${env:cniVersion}.tgz; \
+  tar -xf cni.tgz; \
+  rm cni.tgz
+
+RUN curl.exe -Lo C:\windows\wins.exe https://github.com/rancher/wins/releases/download/v0.0.4/wins.exe
+
+RUN curl.exe -Lo C:\windows\yq.exe https://github.com/mikefarah/yq/releases/download/2.4.1/yq_windows_amd64.exe
```

Comment:
i don't think we should install wins.exe and yq.exe to c:\windows. the system root is for windows originated software.

Reply:
Yeah, I was just being lazy since it's one of the few places that's on the PATH by default. I will update to somewhere else.